pkg/virtctl/vm/vm.go

```diff
@@ -323,105 +97,6 @@ func usage(cmd string) string {
 	return usage
 }
 
-func usageExpand() string {
```

Comment:
The generic `usage` func above does not seem very helpful. Can you improve it in a follow up?